tests/conftest.py

```diff
@@ -555,21 +573,41 @@ def fixture_rnafusion_analysis_dir(analysis_dir: Path) -> Path:
     return Path(analysis_dir, "rnafusion")
 
 
-@pytest.fixture(name="sample1_cram")
-def fixture_sample1_cram(mip_dna_analysis_dir: Path) -> Path:
-    """Return the path to the cram file for sample 1."""
+@pytest.fixture(name="sample_cram")
+def fixture_sample_cram(mip_dna_analysis_dir: Path) -> Path:
+    """Return the path to the cram file for a sample."""
     return Path(mip_dna_analysis_dir, "adm1.cram")
 
 
+@pytest.fixture(name="father_sample_cram")
+def fixture_father_sample_cram(mip_dna_analysis_dir: Path) -> Path:
+    """Return the path to the cram file for the father sample."""
+    return Path(mip_dna_analysis_dir, "adm2.cram")
```

Comment:
```suggestion
    return Path(mip_dna_analysis_dir, father_sample_id + FileExtensions.CRAM)
```
You will have to add FileExtensions.CRAM since it does not exist yet